README.md

```diff
@@ -1,2 +1,70 @@
+        file_system_description  = "${var.file_system_description}"
+        access_group_description = "${var.access_group_description}"
+        name        = "${var.name}"
+        type        = "${var.type}"
+        access_group_name = "${module.access_group.this_name}"
+        source_cidr_ip    = "${var.source_cidr_ip}"
+        rw_access_type    = "${var.rw_access_type}"
+        user_access_type  = "${var.user_access_type}"
+        priority          = "${var.priority}"
+        mount_target_domain = "${module.mount_target.this_mount_target_domain}"
+        vswitch_id       = "${var.vswitch_id}"
+	}
+	```
+2. Setting `access_key` and `secret_key` values through environment variables:
+
+	- ALICLOUD_ACCESS_KEY
+	- ALICLOUD_SECRET_KEY
+	- ALICLOUD_REGION
+
+# Input
+
+| Name            | Description                                                  | Type   | Default | Required |
+| :-------------- | :----------------------------------------------------------- | ------ | ------- | -------- |
+| protocol_type	| Type of protocol used. Currently includes the NFS type and the SMB type | string  | Performance | yes |
+| storage_type	| The file system type. Currently includes the Performance type and the Capacity type | string | NFS | yes |
+| description	| description (space characters are not allowed). | string | terraform-alicloud-nas-file-system| no |
+| name		| Permission group name | string | terraform-alicloud-nas-access-group | yes      |
+| type		| Permission group type, including the Vpc and Classic types | string   | Vpc    | yes     |
+| access_group_name | The Name of the access group name.                     | string | null    | yes      |
+| source_cidr_ip    | Address or address segment.                                | string | 168.1.1.0/16    | yes      |
+| rw_access_type    | Read-write permission type: RDWR (default), RDONLY.        | string | RDWR    | no       |
+| user_access_type  | User permission type: no_squash (default), root_squash, all_squash. | string | no_squash | no |
+| priority          | Priority level. Range: 1-100. Default value: 1.            | Integer | 1      | no |
+| vswitch_id        | The ID of the VSwitch ID.                                  | string | vsw-2ze579zqzb2k91twcz7wj | no       |
+| mount_target_domain | The mount_target_domain of the MountTargetDomain.        | string | null    | no       |
+| create_access_group | Determine whether a permission group exists.             | bool   | false   | no       |
+| create_access_rule  | Judging whether permission rules exist.                  | bool   | false   | no       |
+
+
+## Output
+
+| Name            | Description                                                  |
+| :-------------- | :----------------------------------------------------------- |
+| this_file_system_id | The ID of the file system.                               |
+| this_mount_target_domain | The Mount Target Domain of the mount target.        |
```

Comment:
I think the output needs more parametes, like group id, group name, rule id, rule name and so on. Please make all modules' output be this module output.